### ISSUE TITLE
Mobile A11y Testing 11/1

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-dropdown#35]
On Android with Talkback, I don't get any sort of spoken alert that menu is showing, and that it has X items. I only hear the spoken result of the first list item.  This means the user is missing context.
